### PR TITLE
Milestone 1: Basic Project structure

### PR DESCRIPTION
**Pull request by guillainbisimwa**
In this Milestone, I have:

- Used create-react-app to initialize a React app,
- Installed redux and PropTypes dependencies using npm,
- Deployed the application to Heroku (https://react-bookstore-g.herokuapp.com/),
- Added GitHub workflow linters, ESlint, and Stylelints.

### CODE REVIEW

**Review by krys2fa — CHANGES_REQUESTED** [review]
# Status: Marked as Invalid! ⚠️  
Please note that your Code Review Request is being marked as invalid. 
This is because the repository is not professional as a result of not following the gitflow guidelines in this milestone.

Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### STATUS: INVALID PR ❌

This PR has been marked as invalid.

Reason: 
- Please create a development branch from the main branch and raise the pull request from the setup branch to development not from setup branch to main branch as you did here.

Feel free to contact me on slack(@michgolden) if you need further clarifications.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import './index.css';
+
+class Bookstore extends React.Component {
+  // eslint-disable-next-line class-methods-use-this
+  render() {
+    return (
+      <div>Bookstore</div>
+    );
+  }
+}
```
Comment (bellom) [inline src/index.js:12]: I believe you're aware you can't have this Bookstore component in here.... 😄

**Review by bellom — APPROVED** [review]
## Hi Team :clap:

### Voila! You have finished your first React/Redux Project: Milestone 1.

##### You have done wonderfully well, you passed all checklist for this milestone :+1:
 
# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 

**Heads-Up:** _Make sure to read the instructions on pathwright carefully before submitting CRR for milestone 2, that way, you will have a high chance of getting your CRR reviewed just once without submitting it for re-review_

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**